skillmap/src/lib/skillMapParser.ts

```diff
@@ -317,22 +366,43 @@ function inflateMapReward(section: MarkdownSection, base: Partial<MapRewardNode>
     if (section.listAttributes?.["rewards"]) {
         const parsedRewards: MapReward[] = [];
         const rewards = section.listAttributes["rewards"];
-        for (const reward of rewards) {
-            let [kind, ...value] = reward.split(":");
-
-            switch (kind) {
-                case "certificate":
-                    parsedRewards.push({
+        for (const reward of rewards.items) {
+            if (typeof reward === "string") {
+                let [kind, ...value] = reward.split(":");
+
+                switch (kind) {
+                    case "certificate":
```

Comment:
do we need to support this case? the only map that could've been using it (before this current pr changed the syntax) was our own hour of code, right?

Reply:
there are a few right now, i can remove this once release shenanigans are settled but best to keep it for now

Reply:
sounds good! lgtm otherwise